README.md

```diff
@@ -128,12 +181,10 @@ remote:
 To github.com:mab27/Apstra_IBA_Probes.git
    45100ac..55e363d  main -> main
 ```
-7) Create a Pull Request againt `main`. branch.
+6) Create a Pull Request against `main` branch.
 
-<br>
-
-## High-Level Architecture of IBA (WIP)
-Explain the notion of Probe, Collector, Grpah Query ..
+## High-Level Architecture of IBA
+Explain the notion of Probe, Collector, Graph Query ..
```

Comment:
I think this part should just include in the future a hyperlink to the document that you are working on and that will be published as a user guide. We can keep this as is for now, but since there is that effort to write that document we will not duplicate it here, but just hyperlink it. Probably nothing to change at this time, this is more of an FYI

Reply:
Done - Hidden in comments to link to IBA Doc when complete